web/src/views/Repos.vue

```diff
@@ -8,10 +8,14 @@
       <Button :to="{ name: 'repo-add' }" start-icon="plus" :text="$t('repo.add')" />
     </template>
 
-    <div class="space-y-4">
-      <ListItem v-for="repo in searchedRepos" :key="repo.id" :to="{ name: 'repo', params: { repoId: repo.id } }">
-        <span class="text-wp-text-100">{{ `${repo.owner} / ${repo.name}` }}</span>
-      </ListItem>
+    <div class="flex flex-col gap-4">
+      <div class="gap-4 grid grid-cols-1 md:grid-cols-2 lg:grid-cols-2">
+        <RepoCard v-for="repo in repoListAccess" :key="repo.id" :repo="repo" />
```

Comment:
While testing this it was quite confusing to me as repos where duplicated in both lists and I was not able to search in the upper one with the searchable items being quite far down:

![image](https://github.com/user-attachments/assets/365955fc-0afe-4ebe-ab10-e5aede987e8b)

We should probably decide for one list that might be sorted by both

Reply:
Just tried the spacing gap-4 spacing for testing wasn't intended to be pushed.

Reply:
Personally, I also don't like the two columns layout if we use it for a huge list. Its hard to read and cant be caught easily by eye. It can be used for a short list e.g. pinned repos on GitHub or similar but not for such long repo lists.

Reply:
> The idea to keep both lists was:

That makes sense.

> the last four accessed repos

Did I removed that, as I was getting a longer list?

> don't filter them by search

It feels a bit strange seeing them when searching.

Maybe small titles like: `Last used` and `All repos` might help 🤔

Reply:
> Did I removed that, as I was getting a longer list?

Also possible I have never implemented it :D at least that was what I wanted to do

> Maybe small titles like: Last used and All repos might help 🤔

Yes, good point, that should help 👍 

Do you want to take care of finishing this PR, or should I?

Reply:
I am experimenting a bit right now. My current favorite is sth like the following. I would actually favorite to show only the pipeline message without any further context. One other thing I have to check is if we can update the `lastPipelineItem` when receiving updates to its state (or even showing the new latest pipeline in case we receive such an event).

![image](https://github.com/user-attachments/assets/671051e6-6209-49b6-bb2f-2fcd8be40703)

Reply:
I liked the context information but fine for me as well.

> I am experimenting a bit right now.

Ok then I'll leave this PR to you. Ping me if you want me to review again.